### DISCUSSION TITLE
GIA on M1 iMac running Big Sur 11.5.1

### DISCUSSION THREAD

**Discussion by rabrowne** [Ascoware/get-iplayer-automator#346] [Q&A]
I am currently setting up a new M1 iMac and failing to open the latest version of GIA (v1.21.11). When trying to run the app I have received this failure window. Do I need to change some other settings in the iMac to get this to run, please?

“Get iPlayer Automator” can’t be opened because Apple cannot check it for malicious software.
This software needs to be updated. Contact the developer for more information.

Safari downloaded this file today at 2:28 pm from github-releases.githubusercontent.com.

Thanks,
Richard Browne

**Comment by skovatch** [comment]
Oh, I think I know what happened... I allowed my 'company' account to expire and went back to my personal account. I guess I need to re-release with the new certificate.

**Reply by rabrowne** [reply]
Thanks, hoped it would be a simple explanation.

Thank you for the great work you do to help so many of us access our homeland ;-)

**Comment by rabrowne** [comment]
All working fine now, thank you.

**Reply by Cessnafan152** [reply]
The can't be opened etc message happens a lot when you download non-Apple software. Simply right-click on the app in Applications (or wherever you put it) and then open.
